test/parallel/test-dotenv-edge-cases.js

```diff
@@ -35,4 +35,29 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 0);
   });
 
+  it('should not override existing environment variables', async () => {
+    const code = `
+      require('assert').strictEqual(process.env.BASIC, 'existing');
+    `.trim();
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ `--env-file=${validEnvFilePath}`, '--eval', code ],
+      { cwd: __dirname, env: { BASIC: 'existing' } },
+    );
+    assert.strictEqual(child.stderr, '');
+    assert.strictEqual(child.code, 0);
+  });
+
+  it('should override NODE_OPTIONS', async () => {
```

Comment:
No, I don't think it should.

Reply:
Can you please explain why?

Reply:
Because all environment variables should be treated the same way. The docs don't mention treating any as an exception. I don't know why we should.